src/registrar/assets/js/get-gov.js

```diff
@@ -209,10 +206,31 @@ function handleInputValidation(e) {
 function handleValidationClick(e) {
   const attribute = e.target.getAttribute("validate-for") || "";
   if (!attribute.length) return;
+
   const input = document.getElementById(attribute);
   runValidators(input);
 }
 
+
+function handleFormsetValidationClick(e) {
+  // Check availability for alternative domains
+  
+  const alternativeDomainsAvailability = document.getElementById('check-avail-for-alt-domains');
+  
+  // Collect input IDs from the repeatable forms
+  let inputIds = Array.from(document.querySelectorAll('.repeatable-form input')).map(input => input.id);
+
+  // Run validators for each input
+  inputIds.forEach(inputId => {
+    const input = document.getElementById(inputId);
+    runValidators(input);
+  });
+
+  // Set the validate-for attribute on the button with the collected input IDs
+  // Not needed for functionality but nice for accessibility
+  alternativeDomainsAvailability.setAttribute('validate-for', inputIds.join(', '));
```

Comment:
```suggestion
  inputIds = inputs.map(input => input.id).join(', ')
  alternativeDomainsAvailability.setAttribute('validate-for', inputIds);
```
In following the example above, you would need to do this here